### PR TITLE
#164393448 Configure eslint using airbnb config

### PR DESCRIPTION
**Pull request by pmndifon**
#### What does this PR do?
- Add eslint airnbnb configuration

#### Description of Task to be completed?
- Install eslint, eslint-config-airbnb-base and eslint-plugin-import

#### How should this be manually tested?
- Not yet implemented

#### What are the relevant pivotal tracker stories?
#164393448

#### Screenshots (if appropriate)
none

#### Questions:
How do i add an eslint script to package.json

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+module.exports = {
+    "root": true,
+    "env": {
+      "browser": true,
+      "es6": true,
+      "node": true,
+    },
+    "extends": ["airbnb", "eslint:recommended"],
```
Comment (jundelikwo) [inline .eslintrc.js:8]: @pmndifon Change it to this
`"extends": ["eslint:recommended","airbnb"],`

**Review by jundelikwo — CHANGES_REQUESTED** [review]

**Review by jundelikwo — APPROVED** [review]
Excellent work @pmndifon

**Review by davidshare — APPROVED** [review]
Well done Patrick

**Review by Danielshow — APPROVED** [review]
Nice one @pmndifon, LGTM

**Pull request merged**